packages/drivers/form/src/form.js

```diff
@@ -1,5 +1,6 @@
 import { types } from 'k-ramel'
 import actions from './actions'
+import bulkActions from './bulk.actions'
```

Comment:
I think `batch` it's more explicit , no ?

Reply:
We use both terms in common dev API

Reply:
As you want ;)